src/curve_fit.jl

```diff
@@ -44,7 +44,7 @@ function lmfit(f, p0, wt; autodiff = :finite, kwargs...)
     lmfit(R, p0, wt; kwargs...)
 end
 
-function lmfit(R::OnceDifferentiable, p0, wt; autodiff = :finite, kwargs...)
+function lmfit(R::OnceDifferentiable, p0, wt; autodiff = :finite, inplacejac = false, kwargs...)
```

Comment:
What's the significance of adding this?

Reply:
Oops it's a remain of when I was still trying to make it work, indeed it doesn't make sense there and should be removed